src/Utilities/Compiler/Options/EditorConfigOptionNames.cs

```diff
@@ -229,5 +229,10 @@ internal static partial class EditorConfigOptionNames
         /// String option to configure names of additional "None" enum case (separated by '|') for CA1008.
         /// </summary>
         public const string AdditionalEnumNoneNames = "additional_enum_none_names";
+
+        /// <summary>
+        /// Boolean option whether to skip analysis when the assembly exposes its internals.
+        /// </summary>
+        public const string SkipWhenInternalsVisibleIsPresent = "skip_when_internalsvisibleto_is_present";
```

Comment:
The option naming and semantics feels a bit inverted, a _skip_ option which is `true` by default sounds a bit weird. I don't have better naming suggestion for negating this option semantics though. @buyaa-n @stephentoub, thoughts? I am fine with retaining your current name and semantics if Buyaa and Stephen don't share my feeling.

Reply:
We could swap the polarity and have `ignore_internalsvisibleto` which would default to `false`.

Reply:
Done.